ofmt/package.json

```
@@ -7,20 +7,21 @@
     "olint": "./bin/olint.js"
   },
   "dependencies": {
+    "@ottofeller/eslint-config-ofmt": "1.1.5",
     "@ottofeller/eslint-plugin-ottofeller": "0.1.3",
     "@typescript-eslint/eslint-plugin": "5.10.2",
     "@typescript-eslint/parser": "5.10.2",
     "eslint": ">= 8",
+    "eslint-plugin-eslint-comments": "3.2.0",
     "eslint-plugin-import": "2.25.4",
     "eslint-plugin-react": "7.28.0",
     "eslint-plugin-react-hooks": "4.3.0",
     "eslint-plugin-tailwindcss": "3.4.3",
-    "meow": "10.1.2"
+    "meow": "10.1.2",
+    "prettier-plugin-organize-imports": "^2.3.4"
```

@quesabe let's freeze the version:
```suggestion
    "prettier-plugin-organize-imports": "2.3.4"
```

---

By the way, is it enough to only add the plugin in the deps?